entrypoint.sh

```diff
@@ -13,11 +13,11 @@ fi
 
 if [ -z "API_SERVICE_URL" ]
 then
-  echo "{'ts': ${date}, 'message': {'text: 'You have not specified an API_SERVICE_URL', 'level': 'ERROR'}}" >&2
+  echo "{'ts': '`date -Iseconds`', 'message': {'text: 'You have not specified an API_SERVICE_URL', 'level': 'ERROR'}}" >&2
```

Comment:
Specified format give timezone offset.
Replace with the following:
`date -u +%FT%T.%3NZ`